TypeScript error: 'cache' is not exported from 'next/cache' in fetchOwner.tsx

## Issue Summary

TypeScript error in `apps/web/app/analyze/(org)/[owner]/fetchOwner.tsx` where `cache` is imported from `next/cache` but this module does not export `cache`.

## TypeScript Error

```
app/analyze/(org)/[owner]/fetchOwner.tsx(3,10): error TS2305: Module '"next/cache.js"' has no exported member 'cache'.
```

## Impact

- **Runtime error risk**: The `cache` function is used to wrap `fetchOwnerInfo`, but the import is invalid
- **Build failure**: This error would cause build failures if `ignoreBuildErrors` is disabled
- **Performance**: The caching mechanism is not working as intended

## Current Code

```typescript
import { cache } from 'next/cache';

export const fetchOwnerInfo = cache(async (orgName: string) => {
  // ...
});
```

## Suggested Fix

In Next.js 15, `cache` has been moved. Use one of these approaches:

1. Import from `react`:
```typescript
import { cache } from 'react';
```

2. Or use `unstable_cache` from `next/cache`:
```typescript
import { unstable_cache } from 'next/cache';

export const fetchOwnerInfo = unstable_cache(
  async (orgName: string) => {
    // ...
  },
  ['fetchOwnerInfo'],
  { tags: ['owner'] }
);
```

## Files Affected

- `apps/web/app/analyze/(org)/[owner]/fetchOwner.tsx`

## Priority

**High** - Potential runtime error and broken caching mechanism

## References

- Next.js 15 cache function migration: https://nextjs.org/docs/app/api-reference/functions/cache

--- a/apps/web/app/analyze/(org)/[owner]/fetchOwner.tsx
+++ b/apps/web/app/analyze/(org)/[owner]/fetchOwner.tsx
@@ -1,6 +1,8 @@
 import { getOwnerInfo } from '@/components/Analyze/utils';
 import { notFound } from 'next/navigation';
-import { cache } from 'next/cache';
+// React.cache deduplicates requests within a single render pass (RSC)
+// @ts-expect-error - React 18.2 types don't export cache, but it exists at runtime
+import { cache } from 'react';
 
 export const fetchOwnerInfo = cache(async (
   orgName: string,

